scripts/project.js

```diff
@@ -0,0 +1,53 @@
+// Define the runtime for Regenerator-compiled generator and async functions
+import 'regenerator-runtime/runtime';
+// Import the api object from requests.js
+const api = require('./requests.js');
+// Import the dom object from dom.js
+const dom = require('./dom.js');
+// Define the function used to get URL parameters - this is external code and does not need to be documented
+// The url variable is a string
+let getParams = function(url) {
+  var params = {};
+  var parser = document.createElement('a');
+  parser.href = url;
+  var query = parser.search.substring(1);
+  var vars = query.split('&');
+  for (var i = 0; i < vars.length; i++) {
+    var pair = vars[i].split('=');
+    params[pair[0]] = decodeURIComponent(pair[1]);
+  }
+  return params;
+};
+// Get the project ID from the URL
+let projectId = getParams(window.location.href).id;
+
+document.addEventListener('deviceready', function() {
+  if (device.platform == "Android") {
+    // Change the window.open function to open a Chrome custom tab via Cordova plugin
+    window.open = cordova.plugins.browsertab.openUrl;
+  }
+  api.project(projectId).then((data) => {
+    let projectTitle = data.title.length > 40 ? data.title.substring(0, 40) + '...' : data.title;
```

Comment:
You can do that with just CSS! `text-overflow: ellipsis` or something like that.

Reply:
I'll look at it!

Reply:
Looks like CSS can only do single-line wraps.  I'm going for ~2 lines.

Reply:
You can do two lines ifn you correctly set the height...

Reply:
?

Reply:
Did you fix this, @micahlt ?

Reply:
What the hell...

Reply:
I give up